### ISSUE TITLE
Release stable version 

### ISSUE THREAD

**Issue by Siedlerchr** [tobiasdiez/EasyBind#76]
Currently, we use  com.tobiasdiez:easybind:2.2.1-SNAPSHOT in JabRef.

I would appreciate it if you could do a release a stable version with the jfx19 fixes.

Background: 
https://discourse.jabref.org/t/cant-build-jabref-reproducibly/3427/11

**Comment by Stewori** [comment]
This issue is over two years old now... maybe at least the maven coordinates for the sonatype snapshot repository used to work back then. This isn't the case any more:
https://oss.sonatype.org/content/repositories/snapshots/com/tobiasdiez/easybind/
is a dead link now. Even without a proper stable release, it would be good to have at least some workable maven coordinates for the 2.2.1 snapshot again.

Edit:
Okay, I see - https://repo1.maven.org/maven2/org/jabref/easybind/2.3.0 is a maintained version on maven central now...

**Comment by Siedlerchr** [comment]
@Stewori  Yes we have a maintained fork at JabRef https://github.com/JabRef/EasyBind
